draft-ietf-quic-transport.md

```diff
@@ -947,47 +947,104 @@ The fields in the ACK frame are as follows:
-    indicates the number of contiguous packets being acked starting after the
-    end of the previous gap.  Along with the previous field, this field is
-    repeated "Num Blocks" times.
+### Ack Block Section {#ack-block-section}
 
-* Timestamp Section:
+The Ack Block Section contains between one and 256 blocks of packet numbers 
+which have been successfully received. If the Num Blocks field is absent, only 
+the First Ack Block length is present in this section. Otherwise, the Num Blocks 
+field indicates how many additional blocks follow the First Ack Block Length 
+field. 
+
+~~~
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|              First Ack Block Length (8/16/32/48)            ...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|  [Gap 1 (8)]  |       [Ack Block 1 Length (8/16/32/48)]     ...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|  [Gap 2 (8)]  |       [Ack Block 2 Length (8/16/32/48)]     ...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+                             ...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|  [Gap N (8)]  |       [Ack Block N Length (8/16/32/48)]     ...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+~~~
+{: #ack-block-format title="Ack Block Section"}
 
-  * Num Timestamps: An unsigned 8-bit number specifying the total number of
-    <packet number, timestamp> pairs following, including the First Timestamp.
+The fields in the Ack Block Section are:
 
-  * Delta Largest Acked (opt): An optional 8-bit unsigned packet number delta
-    specifying the delta between the largest acked and the first packet whose
-    timestamp is being reported.  In other words, this first packet number may
-    be computed as (Largest Acked - Delta Largest Acked.)
+* First Ack Block Length: An unsigned packet number delta that indicates the
+  number of contiguous additional packets being acked starting at the Largest
+  Acked.
 
-  * First Timestamp (opt): An optional 32-bit unsigned value specifying the time
-    delta in microseconds, from the beginning of the connection to the arrival
-    of this packet.
+* Gap To Next Block (opt, repeated): An unsigned number specifying the number
+  of contiguous missing packets from the end of the previous ack block to the
+  start of the next.
+
+* Ack Block Length (opt, repeated): An unsigned packet number delta that
+  indicates the number of contiguous packets being acked starting after the
+  end of the previous gap.  Along with the previous field, this field is
+  repeated "Num Blocks" times.
+
+### Timestamp Section {#timestamp-section}
+
+The Timestamp Section contains between zero and 255 measurements of packet
+receive times. This information can be used by a sender to better estimate the
+RTT of the connection.
```

Comment:
I would leave the second sentence out, because I don't think improving RTT estimates are the most likely use case.

I would suggest adding "relative to the beginning of the connection." to the end of the first sentence to clarify these aren't an effort to convey a true time.